backend/internal/storage/subscribers.go

```diff
@@ -0,0 +1,34 @@
+package storage
+
+import (
+	"github.com/elhmn/jobsika/pkg/models/v1beta"
+	"gorm.io/gorm"
+)
+
+const SubscribersTableName = "job_subscribers"
```

Comment:
If that variable is called subscribersTableName maybe we should call the table subscribers ?

Reply:
Another option could be to name that variable `JobSubscriberTableName` and keep the table name `job_subscribers`. What really matters here is consistency